dc/s2n-quic-dc/Cargo.toml

```diff
@@ -12,13 +12,19 @@ exclude = ["corpus.tar.gz"]
 
 [features]
 default = ["tokio"]
-testing = ["bolero-generator", "s2n-quic-core/testing", "s2n-quic-platform/testing", "tracing-subscriber"]
+testing = [
+    "bolero-generator",
+    "s2n-quic-core/testing",
+    "s2n-quic-platform/testing",
+    "tracing-subscriber",
+]
 tokio = ["tokio/io-util", "tokio/net", "tokio/rt-multi-thread", "tokio/time"]
 
 [dependencies]
 arrayvec = "0.7"
 atomic-waker = "1"
 aws-lc-rs = "1"
+bach = "0.0.10"
```

Comment:
dev-dependency? I don't see any usage outside of testing... although maybe testing is public?

Reply:
yep, you're absolutely right. I will fix that in a follow-up